charts/cryostat/README.md

```diff
@@ -70,6 +70,14 @@ A Helm chart for deploying [Cryostat](https://cryostat.io/) on Kubernetes and Op
 | `datasource.resources`        | Resource requests/limits for the JFR Data Source container. See: [ResourceRequirements](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#resources)                                                                                                                                     | `{}`                              |
 | `datasource.securityContext`  | Security Context for the JFR Data Source container. Defaults to meet "restricted" [Pod Security Standard](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted). See: [SecurityContext](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1) | `{}`                              |
 
+### Authentication
+
+| Name                                  | Description                                                                                                                                                                                                                                                           | Value   |
+| ------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| `authentication.basicAuth.enabled`    | Whether Cryostat should use basic authentication for users                                                                                                                                                                                                            | `false` |
```

Comment:
I think it should be made clear that if this is not enabled then the Cryostat instance will use *no* authentication. Otherwise, I worry that people might get confused and expect it to delegate to the OpenShift auth server like what happens by default when using the Operator.

Reply:
Updated! How about now?